packages/typir/src/kinds/kind.ts

```diff
@@ -16,8 +16,10 @@ export interface Kind {
 
     getUserRepresentation(type: Type): string;
 
-    // assumption: both types habe the same kind and this kind owns the called function
+    /** If the kinds of super type and sub type are different, this function will be called for both kinds in order to check,
+     * whether at least one kinds reports a sub-type-relationship. */
     isSubType(superType: Type, subType: Type): TypirProblem[];
```

Comment:
Switch the positions of super and sub. It reads like "super is sub type of sub".

Reply:
Let's directly discuss that later

Reply:
done